lib/modules/platform/gerrit/index.ts

```diff
@@ -0,0 +1,544 @@
+import JSON5 from 'json5';
+import type { RepoGlobalConfig } from '../../../config/types';
+import { logger } from '../../../logger';
+import type { BranchStatus } from '../../../types';
+import * as git from '../../../util/git';
+import { setBaseUrl } from '../../../util/http/gerrit';
+import { regEx } from '../../../util/regex';
+import { ensureTrailingSlash } from '../../../util/url';
+import type {
+  BranchStatusConfig,
+  CreatePRConfig,
+  EnsureCommentConfig,
+  EnsureCommentRemovalConfigByContent,
+  EnsureCommentRemovalConfigByTopic,
+  EnsureIssueConfig,
+  EnsureIssueResult,
+  FindPRConfig,
+  Issue,
+  MergePRConfig,
+  PlatformParams,
+  PlatformResult,
+  Pr,
+  RepoParams,
+  RepoResult,
+  UpdatePrConfig,
+} from '../types';
+import { repoFingerprint } from '../util';
+
+import { smartTruncate } from '../utils/pr-body';
+import readOnlyIssueBody from '../utils/read-only-issue-body';
+import { client } from './client';
+import { configureScm } from './scm';
+import type {
+  GerritChange,
+  GerritFindPRConfig,
+  GerritLabelTypeInfo,
+  GerritProjectInfo,
+} from './types';
+import {
+  TAG_PULL_REQUEST_BODY,
+  getGerritRepoUrl,
+  mapBranchStatusToLabel,
+  mapGerritChangeToPr,
+  mapPrStateToGerritFilter,
+} from './utils';
+
+export const id = 'gerrit';
+
+const defaults: {
+  endpoint?: string;
+} = {};
+
+let config: {
+  repository?: string;
+  head?: string;
+  config?: GerritProjectInfo;
+  labels: Record<string, GerritLabelTypeInfo>;
+  labelMappings?: {
+    stabilityDaysLabel?: string;
+    mergeConfidenceLabel?: string;
+  };
+  gerritUsername?: string;
+} = {
+  labels: {},
+  labelMappings: {
+    stabilityDaysLabel: 'Renovate-Stability',
+  },
+};
+
+export function mergeToConfig(newConfig: typeof config): void {
+  config = { ...config, ...newConfig };
```

Comment:
this is a flat merge, so `labels` and `labelMappings` will be overwritten and not merged